cmd/release.go

```diff
@@ -232,6 +232,7 @@ func readChangelog() io.ReadCloser {
 }
 
 // getChangelog returns the changelog's header and body for a given version.
+// Returns empty strings if the given version is not found.
 func getChangelog(version string, rc io.ReadCloser) (string, string) {
```

Comment:
can we name return arguments? Also what is `name`? Title of version or what?

Reply:
It's the name of the release, I'll update it to be more explicit.